app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -188,10 +188,17 @@ def async_state_done(current_async_state)
         },
       )
 
-      form_response.extra[:ssn_is_unique] = DuplicateSsnFinder.new(
-        ssn: idv_session.ssn,
-        user: current_user,
-      ).ssn_is_unique?
+      if current_async_state.result[:ssn_is_unique]
+        # new state
+        # It would be cleaner to reverse this, but what about existing logs?
+        form_response.extra[:ssn_is_unique] = current_async_state.result[:ssn_is_unique]
```

Comment:
Based on @matthinz's note in the original ticket that there isn't anything obvious looking at this, I'm tempted to _not_ do this, and just have `result[:ssn_is_unique]` come back and get logged implicitly, rather than doing this to keep perfect backwards compatibility.

Reply:
as long as there's a spec that covers, it makes sense. I saw your comment about testing it being difficult, lmk if you need more eyes on that

Reply:
`#idv_result_to_form_response` takes the result we get from the background job and makes it into the `form_response` we see here. So what this code is doing is taking the value that is on `current_async_state` and using it to re-assign the value that we got from `current_async_state` in `#idv_result_to_form_response`.

I think the idea of this change is that `VerifyInfoConcern` can forget about `ssn_is_unique` all together. We should be able to remove this entire stanza and get the effect we want when `ssn_is_unique` is on the result we get from the job.

Reply:
As proposed, I backed out my changes to this file.